src/configuration/index.ts

```diff
@@ -373,4 +377,39 @@ export default class ConfigurationAPI extends WebAPI {
   async getLicenseWebhooksState(client_id?: string): Promise<WebhooksState> {
     return this.send("get_license_webhooks_state", { client_id });
   }
+
+  /**
+   * Creates an auto access data structure, which is a set of conditions for the tracking URL and geolocation of a customer.
+   * @param opts - options for
+   */
+  async addAutoAccess(opts: AddAutoAccessRequest): Promise<AddAutoAccessResponse> {
+    return this.send("add_auto_access", opts);
+  }
+
+  /**
+   * Returns all existing auto access data structures.
+   */
+  async listAutoAccesses(): Promise<AutoAccess[]> {
+    return this.send("list_auto_accesses", {});
+  }
+
+  /**
+   * Deletes an existing auto access data structure specified by its ID.
+   * @param id - auto access ID
```

Comment:
please follow convention in param description

Reply:
Fixed. Completed the description for parameters.